views/multi_room.haml

```diff
@@ -62,4 +62,4 @@
     findClass[i].style.height = tallest + "px";
   }
 
-  equalHeights('calentry')
```

Comment:
Was this intended to be removed?

Reply:
I intended to remove it, yes. JavaScript is not my area of expertise and I wasn't sure what that was doing there. I asked my mentor and we had a look together and realised that `equalHeights` is not defined anywhere or used in any way and does not change the behaviour of the app in any form, so our conclusion was that probably someone forgot to delete this line time ago. Please, feel free to correct me if I'm wrong